scaffold/convert/header.template.js

```diff
@@ -14,6 +17,18 @@
 %>
   return request;
 };
+<% } else if (customBasic) { %>
+const maybeIncludeAuth = (request, z, bundle) => {
+  const mapping = {
+    username: '<%= mapping.username %>',
+    password: '<%= mapping.password %>'
+  };
+  const username = replaceVars(mapping.username, bundle);
+  const password = replaceVars(mapping.password, bundle);
+  const encoded = `${username}:${password}`.toString('base64');
```

Comment:
`toString()` on a string won't encode in JS like you'd hope. Need to wrapper in a Buffer I believe.

```
> 'asdf:'.toString('base64')
'asdf:'
> new Buffer('asdf:').toString('base64')
'YXNkZjo='
```

Reply:
@bcooksey nice catch! Fixed in 7ac3941.